test/index.js

```diff
@@ -1,22 +1,38 @@
 var assert = require('assert');
-var Cookies = require('js-cookie');
 var mock = require('mock-cookie');
 
-require('../src/index.js');
-
 document = new mock.Document();
 
 describe('Cookies', function () {
   describe('#removeAll()', function () {
     it('should remove all cookies', function () {
+      var Cookies = require('../src/index.js');
+
       Cookies.set('a', '1');
       Cookies.set('b', '2');
       assert.equal('1', Cookies.get('a'));
       assert.equal('2', Cookies.get('b'));
 
       Cookies.removeAll();
-      assert.equal(undefined, Cookies.get('a'));
-      assert.equal(undefined, Cookies.get('b'));
```

Comment:
The documented order of the `assert` package is `(actual value, expected constant)`. Here it is being used `(expected constant, actual value)` instead. I created some variables to make it clear what goes where and document the assert signatures. This is a big problem with assertions like these, it is like a [Boolean Trap](http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html) where you can't know what the argument means without looking up in the docs.